assets/src/edit-story/components/library/panes/shapes/shapePreview.js

```diff
@@ -34,13 +34,28 @@ const PREVIEW_SIZE = 36;
 
 const ShapePreviewContainer = styled.button`
   background: transparent;
-  border: 0;
+  border: 1px solid ${({ theme }) => theme.colors.fg.gray16};
+  border-radius: 4px;
   position: relative;
-  padding: 0.8em 0.5em;
+  margin: 0.8em 0.5em;
   flex: 0 0 25%;
```

Comment:
I assume 25% means there should be 4 shapes in a row (per old design it should be so, couldn't find the new design for that, where could that be?) -- however, currently there's other stuff (such as the scrollbar sometimes / margin) taking room and it ends up being 3 on a row with extra room on the right side:
<img width="359" alt="Screenshot 2020-08-20 at 18 32 48" src="https://user-images.githubusercontent.com/3294597/90792781-3285db00-e30b-11ea-8457-a15691b53cd1.png">

Reply:
Fixed layout in 6008c7504. Used hard pixel values as we don't have proper responsive layout anyway.